### ISSUE TITLE
/E Multi-Mail users defaults to line editor instead of sysop selected text editor

### ISSUE THREAD

**Issue by the-godfather-007** [wwivbbs/wwiv#1653]
/E does not open to FSED if that is the SysOp selection in the defaults menu.  It defaults to line editor.  Works as expected otherwise.  Great add getting this working, thank you.

**Comment by the-godfather-007** [comment]
Does not work even with proposed update from a closed issue marked as resolved.

"probably just changing line 75 from

data.fsed_flags = FsedFlags::NOFSED;

to

data.fsed_flags = FsedFlags::FSED;"

Did not work for me on Linux.

**Comment by the-godfather-007** [comment]
Also note, since replacing the modded multmail.cpp file pulled from merged update; I can see the AKA's from other posts, while also on taglines I'm seeing a diamond ANSI character which I believe is a heart code.  I don't know what could be causing this.  Other changes I've made are unrelated such as simply restoring default colors for my user back to system colors found within wwiv.ini.

**Comment by ericpareja** [comment]
What FSED are you using?

**Comment by ericpareja** [comment]
Cross referencing #1650

**Comment by the-godfather-007** [comment]
The one built into WWIV with v5.9-dev 
I made the changes from #1650 but that did not work

**Comment by robertstinnett** [comment]
I am on 5.9.03748  and this appears to be working as it should -- have Full Screen Editor selected in Defaults, and it uses it.

**Comment by the-godfather-007** [comment]
Yes this can be closed.  Once  I made the changes, I had to go into defaults and select line editor, exit, then go back in and select FSED, and it began working. Somehow resetting my default this way worked.  Thank you for looking into it.

**Comment by ericpareja** [comment]
Closing as noted in previous comments.
